src/main/java/org/jsoup/parser/Tag.java

```diff
@@ -122,7 +123,7 @@ public boolean canContainBlock() {
      * @return if this tag is an inline tag.
      */
     public boolean isInline() {
-        return !isBlock;
+        return inlineTag;
```

Comment:
Sorry, I don't get this one. No tag is marked as an inlineTag in this impl. https://github.com/jhy/jsoup/blob/7c60ac132afc0034da18bfe969e18046994f9427/src/main/java/org/jsoup/parser/Tag.java#L285

They get marked as !block and !formatBlock. What do we need / use this field for?

Reply:
The original implementation of `isInline()` returned true for non-inline tags (e.g.: `<p>`) also.
That's why I made it stricter.

In the meantime I've noticed that happened because `<p>` is included in the `formatAsInlineTags` list.
https://github.com/jhy/jsoup/blob/7c60ac132afc0034da18bfe969e18046994f9427/src/main/java/org/jsoup/parser/Tag.java#L263
Would the desired behavior be to skip line breaks for inline tags and those listed in `formatAsInlineTags`?